### PR TITLE
Datepicker: removed arrow symbols from next and previous month hovertext in swedish

### LINKED ISSUE #2048: Datepicker, Next month button tooltip
In some languages there is some text after the word "Next" in whatever language. In my case it is swedish and the tooltip is this: Nästa&#xBB;

Added a picture of the same problem in chinese and for replication you can check here:
https://jqueryui.com/datepicker/#localization

d3dfef2e-09ff-4d3d-af6a-4cc56dee3ed3

### PR DESCRIPTION
**Pull request by adalid-1**
Fixes #2048

### CODE REVIEW

**Comment by adalid-1** [comment]
Commit is made from work git account and PR is made with private git account.

**Comment by mgol** [comment]
@fnagel any idea why these symbols were included in some language versions, leading to issues like #2048, and not in others, like en-GB?

**Comment by adalid-1** [comment]
@mgol I rolled back the local branch and recomitted the change with correct author and did a forced push. I am not great with git so please be wary if you see something wierd. If so I can remake the pullrequest from scratch after the weekend.

**Comment by mgol** [comment]
Yeah, I'd also opt into removing all of those prefixes/suffixes. @adalid-1 would you like to do it for all the locales?

**Comment by adalid-1** [comment]
@mgol Sure thing!

File: ui/i18n/datepicker-bs.js [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
 datepicker.regional.bs = {
 	closeText: "Zatvori",
-	prevText: "&#x3C;",
-	nextText: "&#x3E;",
+	prevText: "",
```
Comment (adalid-1) [inline ui/i18n/datepicker-bs.js:20]: hmm

File: ui/i18n/datepicker-hr.js [hunk 2]
```diff
@@ -17,8 +17,8 @@
 
 datepicker.regional.hr = {
 	closeText: "Zatvori",
-	prevText: "&#x3C;",
-	nextText: "&#x3E;",
+	prevText: "",
```
Comment (adalid-1) [inline ui/i18n/datepicker-hr.js:20]: empty

File: ui/i18n/datepicker-mk.js [hunk 3]
```diff
@@ -17,8 +17,8 @@
 
 datepicker.regional.mk = {
 	closeText: "Затвори",
-	prevText: "&#x3C;",
-	nextText: "&#x3E;",
+	prevText: "",
```
Comment (adalid-1) [inline ui/i18n/datepicker-mk.js:20]: empty

**Comment by adalid-1** [comment]
A few languages turned up empty when removing arrow. Maybe that is ok?

**Comment by adalid-1** [comment]
Do I need to do anything more here or can I just wait for someone to review and merge this?

**Comment by mgol** [comment]
Hmm, I wonder if this is really behaving in the same way as 1.12.1 did. As `prevText` & similar were documented to accept text, not HTML but were in fact accepting HTML, we switched that to only accept text as a security fix. It looks like this is something that broke those arrows... @adalid-1 are you sure the behavior in 1.12.1 was the same?

Making the texts empty may be bad for accessibility, potentially worse than at least having an arrow (as long as it works, of course). Maybe we should leave the arrows - but fixed, inserted as Unicode, not HTML entities - in those languages that would otherwise have empty `prevText`, etc.? What do you think, @fnagel?

**Comment by adalid-1** [comment]
You are right it works in the older version. I was misstaken, I must have made some changes that did not come through when testing earlier. Now that i tested downloading the older version from here:
https://jqueryui.com/download/
And setting nextText to contain the arrow it came out as intended.

**Comment by fnagel** [comment]
> As prevText & similar were documented to accept text, not HTML but were in fact accepting HTML, we switched that to only accept text as a security fix.

Sounds reasonable. Anyway, a fix would be good.


> A few languages turned up empty when removing arrow. Maybe that is ok?
> Making the texts empty may be bad for accessibility, potentially worse than at least having an arrow (as long as it works, of course).
 
We should definitely not have those texts empty. I'm ok with using the arrows but having a real text would be better. I could add a commit introducing those missing texts translated by Google Translate (and verified by another). @mgol What do you think?

**Comment by fnagel** [comment]
We might want to merge #2067 before working on this. This one seems related: #1900

**Comment by mgol** [comment]
This landed in 3126e128691fd9fa78f33930ba9cb37609d0a64f, thanks!

**Pull request closed without merging**